spec/empty_spec.lua

```diff
@@ -0,0 +1,23 @@
+-- .vscode/settings.json <<
```

Comment:
Why add this comment?

Reply:
Sorry forgot to reply 😿 

It seems the original author tend to do so, I believe we could safely remove them if needed